public/templates/public/blocks/featured_library_expert.html

```diff
@@ -23,10 +23,8 @@ <h6>Not sure where to start?</h6>
             <li><a href="{{self.url}}">Ask a Librarian</a></li>
         </ul>
     </div>
-    {% if profile_pic %}
-        <div class="col-xs-4 home-profile">
-            <img src="{{profile_pic.url}}" alt="{{profile_pic.title}}" class="img-responsive" />
-        </div>
-    {% endif %}
+
```

Comment:
If this spacing is showing correctly in git (sometimes it doesn't) we probably don't need this much.

Reply:
Ah, weird.  I'm going to go ahead and revert this to the `master` version for now since we're planning to open up a new issue anyway investigating whether it's getting used.